modules/ROOT/pages/technical-information.adoc

```diff
@@ -34,10 +34,23 @@ whenever you update the base OS image.
 [[filesystem-layout]]
 == Silverblue filesystem layout
 
-On Silverblue, the root filesystem is immutable. This means that `/`, `/usr` 
-and everything below it is read-only.
+On Silverblue, the root filesystem is immutable. This means that the following 
+system directories and everything below them are read-only:
 
-`/var` is where all of Silverblue's runtime state is stored. Symlinks are used 
+* `/`
+* `/usr`
+
+On the otherhand, the following system directories are writable using the live filesystem:
+
+* `/boot`
+* `/dev`
+* `/etc`
+* `/run`
+* `/sysroot`
+* `/tmp`
+* `/var`
```

Comment:
There's an effort to make `/boot` and `/sysroot` read-only too (not sure if it made it to silverblue yet). I wouldn't mention `/dev/`. It's not really a filesystem that you would store files in. Might be worth mentioning that `/run/` and `/tmp/` are tmpfs (i.e. just files in RAM).

Reply:
I incorporated the suggestions.